src/components/table-of-contents/toc.spec.js

```diff
@@ -52,10 +52,120 @@ describe('script: table-of-contents', () => {
+                    <p>Census Day was on Sunday 21 March 2021.</p>
+
+                    <p>If you still have a paper census questionnaire, fill it in as soon as you can and return it to FREEPOST, Census 2021.</p>
+                </section>
+                <section id="section3">
+                    <h2>What happens after Census Day</h2>
+                    <p>Two follow-up surveys, the <a href="#0">Census Coverage Survey</a> and the <a href="#0">Census Quality Survey</a>, help to improve the quality and accuracy of the census results.</p>
+
+                    <p>Over the next year, we work hard to process the answers from all the census questionnaires. This is so that we can publish the initial Census 2021 findings one year after the census and the main releases two years after the census.</p>
+
+                    <p>To make sure the Census 2021 statistics we publish meet the needs of those who use them, we also run an outputs consultation.</p>
+                </section>
+                <section id="section4">
+                    <h2>The census in Northern Ireland and Scotland</h2>
+                    <p>The Office for National Statistics (ONS) runs the census in England and Wales.</p>
+
+                    <p>If you live in Northern Ireland, visit the Northern Ireland Statistics Agency (NISRA) census website to find out about the census for Northern Ireland.</p>
+
+                    <p>If you live in Scotland, visit the National Records of Scotland census website to find out how to take part in the census for Scotland.</p>
+                </section>
+                <section id="section5">
+                    <h2>The last census</h2>
+                    <p>The last census took place in 2011. Many people and organisations used information from the 2011 Census in a variety of ways.</p>
+
+                    <p>For example, Bristol City Council used it to inform decisions on how to fund local housing improvements. It was also essential for the charity Redbridge Council for Voluntary Services to help people from ethnic minority groups learn more about dementia.</p>
+
+                    <p>To read more about the benefits that information from the 2011 Census helped to create, visit the Office for National Statistics (ONS) website.</p>
+                    <p>The Census Coverage Survey (CCS) was a short, separate survey that we at the Office for National Statistics (ONS) started six to eight weeks after Census Day.</p>
+
+                    <p>The CCS helps make sure everyone is counted and our census results are as accurate as possible. It asked similar questions to the main census, just fewer of them, at addresses in a selection of postcodes across England and Wales.</p>
+
+                    <p>The CCS has now closed.</p>
+                </section>
+            </div>
+        </div>
+    </div>
+    `,
+      'main',
+    );
+  });
+
+  it('renders a fixed table of contents in full page and scrolls', async () => {
```

Comment:
This line should change to make the description make more sense. Maybe something like, "when the ons-grid__col--sticky is used table of contents is rendered in a fixed position staying in view on the page when scrolling to the bottom"

Reply:
Understood the sentiment, however I think that test descriptions should be kept to within 70 characters for conciseness and clarity. How about "when ons-grid__col--stick is used, ToC stays fixed while scrolling"

Reply:
I understand that, but we haven't yet discussed or set any char limits or standards for our test descriptions. To be fair that is a bit better but I don't think thats quite descriptive enough which I would say is more important that keeping it under 70 characters. If you do want to keep it that short then you could break off the when bit into a `describe` but really that is used for grouping which isn't really needed here. We have a card to look into rewriting our test descriptions anyway. I do prefer something like "when the "ons-grid__col--sticky" class is used, then the ToC stays in a fixed position while scrolling" because its more explicit.